htdocs/web_portal/views/ngi/view_ngi.php

```diff
@@ -216,6 +216,7 @@
                             if ($params['authenticated']) {
                             ?>
                                 <a href="index.php?Page_Type=User&amp;id=<?php echo $role->getUser()->getId() ?>">
+                                    <img src="<?php echo \GocContextPath::getPath()?>img/person.png" height="25px" style="vertical-align: middle; padding-right: 1em;" />
```

Comment:
What is the reasoning behind adding this icon next to each name?

Reply:
I think it was to make it consistent with the pages for service groups which already used icons.